coprocessor/fhevm-engine/fhevm-engine-common/src/healthz_server.rs

```diff
@@ -137,6 +157,7 @@ impl<S: HealthCheckService + Send + Sync + 'static> HttpServer<S> {
 #[derive(Clone, Default)]
 pub struct HealthStatus {
     checks: HashMap<&'static str, bool>,
+    is_dependency_check: HashMap<&'static str, bool>,
```

Comment:
What's the meaning of is_dependency_check?

Reply:
If this is different from what the spec says, maybe add some comments to explain its meaning.